src/loaders/lottie/tvgLottieParser.cpp

```diff
@@ -42,7 +42,10 @@ static char* _int2str(int num)
 
 CompositeMethod LottieParser::getMaskMethod(bool inversed)
 {
-    switch (getString()[0]) {
+    auto mode = getString();
+    if (!mode || strlen(mode) < 1) return CompositeMethod::None;
```

Comment:
please remove the following condition if it doesn't helpful for the fix. that looks unnecessary.
```
 strlen(mode) < 1
```